### PR TITLE
UI: add links to Tab, DefaultCell and BlocksCell

### PR DESCRIPTION
**Pull request by sofa-tata**

### CODE REVIEW

**Comment by sofa-tata** [comment]
checked in both: gohome and wekapp

File: src/components/Table/cells/BlocksCell/BlocksCell.tsx [hunk 1]
```diff
@@ -25,24 +28,37 @@ function BlocksCell<Data extends Record<string, unknown>>({
...
-      <div className='blocks-wrapper'>
-        {value.map(({ uid, id, status }: CellValue) => {
-          const classes = classNames({
-            block: true,
-            [status]: true
-          })
-
-          return <div key={uid ?? id} className={classes} />
-        })}
+  const getCellContent = () => {
```
Comment (mvoloshin0458) [inline src/components/Table/cells/BlocksCell/BlocksCell.tsx:31]: no need to wrap in a function

File: src/components/Table/cells/DefaultCell/DefaultCell.tsx [hunk 2]
```diff
@@ -27,11 +29,23 @@ function DefaultCell({ cell }: CustomCellProps) {
     }
   }, [value])
 
-  return (
-    <Tooltip data={tooltip?.toString()}>
+  const getCellContent = () => {
```
Comment (mvoloshin0458) [inline src/components/Table/cells/DefaultCell/DefaultCell.tsx:32]: probably can be done without a function too

File: src/components/Table/cells/DefaultCell/DefaultCell.tsx [hunk 3]
```diff
@@ -27,13 +29,20 @@ function DefaultCell({ cell }: CustomCellProps) {
...
+      </Link>
+    ) : (
       <div className='table-default-cell' ref={ref}>
         {value}
       </div>
-    </Tooltip>
-  )
+    )
+
+  return <Tooltip data={tooltip?.toString()}>{cellContent}</Tooltip>
```
Comment (mvoloshin0458) [inline src/components/Table/cells/DefaultCell/DefaultCell.tsx:45]: do we need a variable here?
```suggestion

  return <Tooltip data={tooltip?.toString()}>{
  isLink && getUrl ? (
      <Link to={getUrl(row.original)} className='table-link'>
        <div className='table-default-cell' ref={ref}>
          {value}
        </div>
      </Link>
    ) : (
      <div className='table-default-cell' ref={ref}>
        {value}
      </div>
      }
      </Tooltip>
```

Comment (sofa-tata) [reply src/components/Table/cells/DefaultCell/DefaultCell.tsx]: I would still vote for a variable, but remade it it a bit

**Review by mvoloshin0458 — APPROVED** [review]

**Pull request merged**